src/js.rs

```diff
@@ -133,26 +133,42 @@ pub async fn doc(
   print_import_map_diagnostics: bool,
 ) -> anyhow::Result<JsValue, JsValue> {
   console_error_panic_hook::set_once();
-  let root_specifier = ModuleSpecifier::parse(&root_specifier)
-    .map_err(|err| JsValue::from(js_sys::Error::new(&err.to_string())))?;
+  inner_doc(
+    root_specifier,
+    include_all,
+    load,
+    maybe_resolve,
+    maybe_import_map,
+    print_import_map_diagnostics,
+  )
```

Comment:
Maybe we should box this future to avoid stack overflows on Windows? Though I'm not sure we have a codepath in CLI that would hit it

Reply:
This is for the wasm code, so won't affect windows.